app/subscriber/src/components/media-type-filters/FilterOptions.tsx

```diff
@@ -109,12 +109,24 @@ export const FilterOptions: React.FC<IMediaTypeFiltersProps> = ({ filterStoreNam
 
   const getClassName = (type: FilterOptionTypes) => (type === active ? 'active' : 'inactive');
 
+  useEffect(() => {
```

Comment:
Can you provide more information on what this is solving?  Are you essentially waiting for the userinfo to be returned from the API?

You can modify the logic to only make a request when `userInfo` is truthy.  The `useApp()` hook has the `userInfo` state.  You can wait until the `userInfo` is defined before making any requests.